fiat-github/src/main/java/com/netflix/spinnaker/fiat/config/GitHubConfig.java

```diff
@@ -79,13 +74,13 @@ public void log(String message) {
 
   private static class BasicAuthRequestInterceptor implements RequestInterceptor {
 
-    @Getter
     @Setter
     private String accessToken;
 
     @Override
     public void intercept(RequestFacade request) {
-      request.addQueryParam("access_token", accessToken);
+      String encoded = Base64Utils.encodeToString(accessToken.getBytes(StandardCharsets.UTF_8));
+      request.addHeader("Authorization", "Basic " + encoded);
```

Comment:
Added comment - all of these seem to work:

```
curl -u token https://api.github.com/user
curl -u ttomsu:token https://api.github.com/user
curl -u duftler:token https://api.github.com/user
```

Leading me to believe GitHub is stripping the "user:" part of the "user:password" Basic Auth standard.